internal/services/appservice/function_app_flex_consumption_resource_test.go

```diff
@@ -398,6 +401,9 @@ resource "azurerm_function_app_flex_consumption" "test" {
     value = "some-postgresql-connection-string"
     type  = "PostgreSQL"
   }
+  lifecycle {
+    ignore_changes = [webdeploy_publish_basic_authentication_enabled]
+  }
```

Comment:
Can you please elaborate on what is happening here? We shouldn't be ignoring a security setting in the config.

Reply:
This is for testing, I removed it.